addons/website_slides/models/slide_channel.py

```diff
@@ -533,21 +566,12 @@ def _get_categorized_slides(self, base_domain, order, force_void=True, limit=Fal
                 'total_slides': len(category_slides),
                 'slides': category_slides[(offset or 0):(limit + offset or len(category_slides))],
             })
-        # Then add uncategorized slides
-        uncategorized_slides = all_slides.filtered(lambda slide: not slide.category_id)
-        if uncategorized_slides or force_void:
-            category_data.append({
-                'category': False, 'id': False,
-                'name': _('Uncategorized'), 'slug_name': _('Uncategorized'),
-                'total_slides': len(uncategorized_slides),
-                'slides': uncategorized_slides[(offset or 0):(offset + limit or len(uncategorized_slides))],
-            })
+
         return category_data
 
-    def _resequence_slides(self, slide):
+    def _resequence_slides(self, slide, category_id):
```

Comment:
Again, changing method signature in stable is not ok if not retro-compatible.

If you really need it, at least default it `category_id=None` and take a fallback value in the method.
Something like `new_category_id = category_id or slide.category_id.id`